gammapy/stats/tests/test_feldman_cousins.py

```diff
@@ -0,0 +1,35 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+import numpy as np
+from numpy.testing import assert_allclose
+from astropy.tests.helper import pytest
+
+from ...stats import (
```

Comment:
This import doesn't work:
https://travis-ci.org/gammapy/gammapy/jobs/75110654#L1219

There's two options to fix this:

1. Add `from .feldman_cousins import *` to `gammapy/stats/__init__.py`
2. Change the import here to `from ...stats.feldman_cousins import`

Both will work for the tests in Python 2 and 3.

The question is where the end-user should import these function from. `gammapy.stats` or `gammapy.stats.feldman_cousins`?

Currently it looks like we have everything in `gammapy.stats` (see [here](https://github.com/gammapy/gammapy/blob/master/gammapy/stats/__init__.py) and [here](https://gammapy.readthedocs.org/en/latest/stats/index.html#module-gammapy.stats).

So if we follow this for the FC functions also, you should somehow group them, maybe by adding a `fc_` prefix to all of them.

Not sure yet what is best here.
@dlennarz – What do you think / prefer?